api/v1beta2/ibmpowervscluster_types.go

```diff
@@ -92,6 +97,22 @@ type IBMPowerVSClusterSpec struct {
 	CosInstance *CosInstance `json:"cosInstance,omitempty"`
 }
 
+// DHCPServer contains the DHCP server configurations
+type DHCPServer struct {
+	// Optional cidr for DHCP private network
+	Cidr *string `json:"cidr,omitempty"`
```

Comment:
lets add some regex to validate this

Reply:
Yet to find a valid regex, will update soon